lib/rspec/core/formatters/json_formatter.rb

```diff
@@ -6,17 +6,30 @@ module Core
     module Formatters
       # @private
       class JsonFormatter < BaseFormatter
-        Formatters.register self, :message, :dump_summary, :dump_profile, :stop, :close
+        Formatters.register self, :message, :dump_summary, :dump_profile, :stop,
+                            :close, :example_group_started, :example_group_finished
 
         attr_reader :output_hash
 
         def initialize(output)
           super
+          @start = Hash.new(0)
+          @execution_times = Hash.new(0)
           @output_hash = {
             :version => RSpec::Core::Version::STRING
           }
         end
 
+        def example_group_started(group)
+          key =  group.group.metadata[:location]
```

Comment:
I was assuming it's would be easier to debug with id because using string instead of instance it's more familiar to me, but I just tried with the object and it's work really well. I will change to object and update this branch.